### PR TITLE
add myua to vendors list

### PR DESCRIPTION
**Pull request by AntonMsk**
We would like to propose an analytics solution for our customers which supports AMP pages.
We would like to be added to the analytics vendors list.

### CODE REVIEW

File: 3p/integration.js [hunk 1]
```diff
@@ -180,6 +180,7 @@ import {my6sense} from '#ads/vendors/my6sense';
 import {myfinance} from '#ads/vendors/myfinance';
 import {myoffrz} from '#ads/vendors/myoffrz';
 import {mytarget} from '#ads/vendors/mytarget';
+import {myua} from '../ads/vendors/myua';
```
Comment (calebcordry) [inline 3p/integration.js:183]: We no longer need changes in this file. Sorry this likely changed while you were working on this pr please revert.

File: ads/vendors/myua.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/myua.js:2]: nit: 2021

File: ads/vendors/myua.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/myua.md:2]: nit: 2021

**Review by calebcordry — COMMENTED** [review]
Thanks for contributing! Just a couple small comments

**Comment by AntonMsk** [comment]
Thanks for a quick answer! 

Made edits:
- 3p/integration file restored
- copyright year fixed

File: 3p/vendors/myua.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/**
```
Comment (dmanek) [inline 3p/vendors/myua.js:1]: Copyright notices are no longer required. Can you remove them from all files in this PR? Thanks!

Comment (AntonMsk) [reply 3p/vendors/myua.js]: Sure. All cleared

Comment (AntonMsk) [reply 3p/vendors/myua.js]: Sure. All cleared

**Comment by AntonMsk** [comment]
Hi there! Copyrights are cleared. Is there any other suggestions?

**Comment by calebcordry** [comment]
@AntonMsk looks like just a few lint errors. You can find them if you follow the `CircleCI` link above, or you can expose them locally by running `amp lint --local_changes`. Please let me know if you have any issues.

**Comment by AntonMsk** [comment]
Thanks, @calebcordry! I missed that details. 
Looks like only owners approvals left

**Review by calebcordry — APPROVED** [review]

**Comment by calebcordry** [comment]
Thanks for your contribution!

**Pull request merged**